plugins/promdump.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: promdump
+spec:
+  version: v0.2.2
+  homepage: https://github.com/ihcsim/promdump
+  shortDescription: promdump dumps the head and persistent blocks of Prometheus.
+  description: |
+    promdump dumps the head and persistent blocks of Prometheus. It supports
+    filtering the persistent blocks by time range.
+
+    promdump is a tool that can be used to dump Prometheus data blocks. It is
```

Comment:
DISCLAIMER: I'm not very familiar with Prometheus.

I like that this description describes what the plugin actually does. However, when you describe how to use the dumped data, you are giving very detailed instructions (`--web.enable-admin-api`), but still not complete. I think it's a great idea to have a detailed howto to get this job done. But the `description` does not give enough space for that (and would be the wrong place IMO). So what do you think about writing a more elaborate guide and just link to this page from here?

Reply:
> However, when you describe how to use the dumped data, you are giving very detailed instructions (--web.enable-admin-api), but still not complete. 

The intention of that paragraph is to show how promdump is different from the Prometheus `snapshot` endpoint which requires Prometheus to be started with the `--web.enable-admin-api` option. Open to suggestion on how to make that clearer.

> So what do you think about writing a more elaborate guide and just link to this page from here?

WDYT of adding a link in the description to the [Getting Started](https://github.com/ihcsim/promdump#getting-started) guide in the README?